### ISSUE TITLE
javax.net.ssl.SSLHandshakeException occurs.

### ISSUE THREAD

**Issue by Daiki-Kawanuma** [ibm-bluemix-mobile-services/bms-pushnotifications-serversdk-java#33]
Since March 1, 2018, IBM Cloud invalidated TLS 1.0, TLS 1.1, and activated TLS 1.2 only. As a result, using this SDK with IBM Java, "javax.net.ssl.SSLHandshakeException" now occurs.

This is discussed at the following URL.
https://issues.apache.org/jira/browse/HTTPCLIENT-1784

Is it possible to repair to explicitly use TLS 1.2?

**Comment by Daiki-Kawanuma** [comment]
This code works for me.
IBM JDK: 1.8
WAS: 17.0.0.2 Liberty

```
SSLContext sslContext = SSLContexts.custom()
        .useProtocol("SSL_TLSv2")
        .build();

CloseableHttpClient httpclient = HttpClients.custom()
        .setSSLContext(sslContext)
        .build();
```

**Comment by techiemanas** [comment]
I am also facing this issue, Using IBM JDK 7, Expecting a quick fix

**Comment by ManavIBM** [comment]
TLS1.2 included as part of latest release 1.5.0.
Closing the issue.
